### ISSUE TITLE
Update command for copying wikipedia data

### ISSUE THREAD

**Issue by andyk** [amplab/training#11]
Under 4. Logging into the Cluster

The answer to the trouble shooting question about what to do if the key has wrong permissions is:

```
chmod 600 ../ampcamp-strata-tutorial.pem
./spark-ec2 -i <key_file> -k <name_of_key_pair> --copy --resume launch ampcamp
```

Should we now change this to the following?

```
./spark-ec2 -i <key_file> -k <name_of_key_pair> copy-data ampcamp
```

**Comment by shivaram** [comment]
I think the first one is better -- The cluster may not have been setup when the key's permission was wrong and the first one makes sure hdfs, spark start up etc.

**Comment by JoshRosen** [comment]
I fixed this in 60f2dc1c1b385298c0e4830a473f909b5d1fbc76.  I changed it to the second line because it turns out that we already have a FAQ entry in the "Launching your own cluster" page that covers the case where the key permissions prevent the launch from succeeding.

**Comment by andyk** [comment]
Oh, ok. great.
